<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelfCheckForm</class>
 <widget class="QWidget" name="SelfCheckForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自检界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_Parent" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>990</width>
     <height>610</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_Top" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>90</y>
      <width>481</width>
      <height>57</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>32</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>设备正在检测...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>130</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>停止检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>130</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>开始检测</string>
    </property>
   </widget>
   <widget class="RoundProgressBar" name="roundProgressBar_DeviceCheck" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="DRoundProgressBar" name="roundProgressBar2" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>160</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_Status" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>865</y>
     <width>971</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>8</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>系统版本:1.0.0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RoundProgressBar</class>
   <extends>QWidget</extends>
   <header>roundprogressbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DRoundProgressBar</class>
   <extends>QWidget</extends>
   <header>control/DRoundProgressBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
